framework/components/ecl-accordions/ecl-accordions.twig

```diff
@@ -0,0 +1,18 @@
+<div class="ecl-accordions" role="tablist" aria-multiselectable="true">
+  <h2 id="accordion-header-1" class="ecl-accordions__header" role="tab" aria-controls="accordion-panel-1" tabindex="0" aria-selected="true" aria-expanded="true">
```

Comment:
I don't like using headings inside components, as we don't know if they will match page hierarchy correctly.
Moreover, as you don't use specific ecl classes for headings, it make no real sense.